policyengine_us/variables/gov/states/mi/tax/income/deductions/retirement/tier_three/ssa_retired/mi_retirement_benefits_deduction_tier_three_ssa_retired_eligible.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class mi_retirement_benefits_deduction_tier_three_ssa_retired_eligible(
+    Variable
+):
+    value_type = int
+    entity = TaxUnit
+    label = "Eligible for the Michigan tier three retirement benefits deduction qualifying SSA & retired"
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-30",  # (9)(c)
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/BOOK_MI-1040.pdf#page=18",
+        "https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits",
+    )
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.deductions.retirement_benefits.tier_three.ssa_retired
+
+        person = tax_unit.members
+
+        retirement_eligible = (
+            person("year_of_retirement", period) <= p.retirement_year
+        ) & (person("year_of_retirement", period) > 0)
+
+        ssa_eligible = (
```

Comment:
more verbose but `ssa_eligible` implies eligibility for an ssa program or through an ssa channel, when it's more the opposite here
```suggestion
        has_retirement_benefits_from_ssa_exempt_employment = (
```